gef.py

```diff
@@ -1259,12 +1259,12 @@ def is_branch_taken(self, insn):
         if mnemo in ("jl", "jnge"): return val&(1<<flags["overflow"])!=val&(1<<flags["sign"]), "S!=O"
         if mnemo in ("jle", "jng"): return val&(1<<flags["zero"]) or val&(1<<flags["overflow"])!=val&(1<<flags["sign"]), "Z || S!=0"
         if mnemo in ("jne", "jnz"): return val&(1<<flags["zero"]) == 0, "!Z"
-        if mnemo in ("jno"): return val&(1<<flags["overflow"]) == 0, "!O"
+        if mnemo in ("jno",): return val&(1<<flags["overflow"]) == 0, "!O"
```

Comment:
`("jno")` is the same as `"jno"` and so `mnemo in x` was doing a substring match instead of a full string match.